compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/GraphqlCodeModifier.java

```diff
@@ -42,12 +42,13 @@ public GraphqlCodeModifier() {
 
     @Override
     public void init(CodeModifierContext modifierContext) {
-        modifierContext.addSyntaxNodeAnalysisTask(new ServiceAnalysisTask(this.modifierContextMap),
-                                                  SyntaxKind.SERVICE_DECLARATION);
+        modifierContext.addSyntaxNodeAnalysisTask(new ServiceDeclarationAnalysisTask(this.modifierContextMap),
+                SyntaxKind.SERVICE_DECLARATION);
```

Comment:
Let's keep the indentation consistent with the other places. Better if you can update the IDE settings to use the existing indentation patterns.

Reply:
Updated